data/Symbeeosis/codedLists.ttl

```diff
@@ -0,0 +1,55 @@
+bdg:AssayCompound a rdfs:Class; rdfs:subClassOf skos:Concept;
+  rdfs:label "Assay compound codelist class".
+<assayCompound> a skos:ConceptScheme;
+  rdfs:label "Assay compound codelist scheme".
+<assayCompound/DPPH> a skos:Concept, bdg:AssayCompound;
+  skos:inScheme      <assayCompound>; skos:topConceptOf <assayCompound>;
+  skos:prefLabel     "DPPH" ;
+  skos:exactMatch       wd:Q5205953 .
+<assayCompound/ABTS> a skos:Concept, bdg:AssayCompound ;
+  skos:inScheme      <assayCompound>; skos:topConceptOf <asseyCompound>;
+  skos:prefLabel     "ABTS" ;
+  skos:exactMatch       wd:Q287582 .
+<asseyCompound/Trolox> a skos:Concept, bdg:AssayCompound ;
+  skos:inScheme      <asseyCompound>; skos:topConceptOf <asseyCompound>;
+  skos:prefLabel     "Trolox" ;
+  skos:exactMatch       wd:Q245489 .
+<asseyCompound/GallicAcid> a skos:Concept, bdg:AssayCompound;
+  skos:inScheme      <asseyCompound>; skos:topConceptOf <asseyCompound>;
+  skos:prefLabel     "Gallic Acid" ;
+  skos:exactMatch       wd:Q375837 .
+<asseyCompound/Quercetin> a skos:Concept, bdg:AssayCompound;
+  skos:inScheme      <asseyCompound>; skos:topConceptOf <asseyCompound>;
+  skos:prefLabel     "Quercetin" ;
+  skos:exactMatch       wd:Q409478 .
+
+<extractionMethod> a skos:ConceptScheme;
+  rdfs:label "Extraction Method codelist scheme".
+bdg:ExtractionMethod a rdfs:Class; rdfs:subClassOf skos:Concept;
+  rdfs:label "Extraction method codelist class".
+<extractionMethod/Ultrasound> a skos:Concept, bdg:ExtractionMethod;
+  skos:inScheme <extractionMethod>; skos:topConceptOf <extractionMethod>;
+  skos:prefLabel "Ultrasound".
+<extractionMethod/Maceration> a skos:Concept, bdg:ExtractionMethod;
+  skos:inScheme <extractionMethod>; skos:topConceptOf <extractionMethod>;
+  skos:prefLabel "Maceration".
+
+<microbialCount> a skos:ConceptScheme;
+  rdfs:label "Microbial Count ranges codelist scheme" .
+bdg:MicrobialCount a rdfs:Class; rdfs:subClassOf skos:Concept;
+  rdfs:label "Microbial Count ranges codelist class" .
+<microbialCount/LT10> a skos:Concept , bdg:MicrobialCount ;
+  skos:prefLabel "< 10";
+  skos:inScheme <microbialCount> ; skos:topConceptOf <microbialCount> .
+
+<grapeVariety> a skos:ConceptScheme;
```

Comment:
Ask the agricultural researchers about the relation between vinestock, variety, and wine.
- maybe a wine can be made from several varieties, so it should say which varieties (and in what proportion?) have gone into making it
- are there more levels, eg sort, varietal...